### PR TITLE
Update Corolla TSS2 Tune

### PR DESCRIPTION
**Pull request by SCshredder17**
updating corolla tss2 tune

### CODE REVIEW

File: selfdrive/controls/lib/latcontrol_indi.py [hunk 1]
```diff
@@ -44,11 +45,16 @@ def __init__(self, CP):
     self.sat_limit = CP.steerLimitTimer
 
     self.reset()
+    
+    @property
+  def outer_loop_gain(self):
```
Comment (ShaneSmiskol) [inline selfdrive/controls/lib/latcontrol_indi.py:50]: Make sure to delete `self.outer_loop_gain = CP.lateralTuning.indi.outerLoopGain` and fix the indentation of property

**Comment by SCshredder17** [comment]
I dont know if i was supposed to hit close pull request

**Comment by SCshredder17** [comment]
Ready to rock

File: cereal/car.capnp [hunk 2]
```diff
@@ -432,10 +432,11 @@ struct CarParams {
   }
 
   struct LateralINDITuning {
-    outerLoopGain @0 :Float32;
-    innerLoopGain @1 :Float32;
-    timeConstant @2 :Float32;
-    actuatorEffectiveness @3 :Float32;
+    outerLoopGainBP @0 :List(Float32);
+    outerLoopGainBP @1 List(Float32);
```
Comment (ShaneSmiskol) [inline cereal/car.capnp:436]: Syntax error here, should be `:List` not `List`

Comment (SCshredder17) [reply cereal/car.capnp]: Updated, thanks

**Comment by SCshredder17** [comment]
Looking good?

**Comment by ShaneSmiskol** [comment]
Can you drive on this branch and make sure it works? Don't have any time to

**Comment by SCshredder17** [comment]
> Can you drive on this branch and make sure it works? Don't have any time to test it

Yes will do tomorrow. Thanks

**Pull request merged**